lib/dynamodb_framework/dynamodb_table_manager.rb

```diff
@@ -364,11 +365,10 @@ def create_global_index(name, partition_key, range_key = nil, read_capacity = 20
           :key_schema => key_schema,
           :projection => {
               :projection_type => :ALL
-          },
-          :billing_mode => billing_mode
+          }
       }
 
-      unless billing_mode == ''
+      unless billing_mode == 'PAY_PER_REQUEST'
```

Comment:
[Minor] This could just be `if billing_mode == 'PROVISIONED'`

Reply:
Addressed.